index.html

```diff
@@ -5,12 +5,228 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <style>
+      body {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between; 
+      align-items: center;
+      margin: 0;
+      font-family: Roboto, sans-serif;
+      min-height: 100vh;
+    }
+
+    .HeadingBox {
+      background-color: #1F2937;
+      padding: 20px;
+      text-align: center;
+      color: #f2f2f2;
+      width: 100vw;
+      flex-direction: row;
+    }
+
+    .HeaderLogo {
+      color: #F9FAF8;
+      font-weight: bold;
+      font-size: 24px;
+      margin-bottom: 10px;
+  }
+
+    .HeaderLink1,
```

Comment:
Is there a reason you are having `HeaderLink1`, `HeaderLink2`, etc? If the header links all have the same styling, you could simply apply this class to all the places you need.  So this could just be `.HeaderLink`

Reply:
ok, thanks